### PR TITLE
UPDATE SJ DOCS

### PR DESCRIPTION
**Pull request by richardmatthewsdev**
Merge Richard's "Supplejack Onboarding Project" documentation (attached) into SJ installation docs.
Review rest of documentation and update/remove as needed
Identify any major re-writes as separate stories

### CODE REVIEW

**Review by yarhtut — APPROVED** [review]

**Comment by yarhtut** [comment]
It would be great if you can link your `SJ Example docs link` on the Main MD file.

File: _posts/2017-12-04-example-project.md [hunk 1]
```diff
@@ -0,0 +1,469 @@
...
+
+The click ‘Create Parser Script’.
+
+You will now have a screen which will allow you to write your parser script for the harvester, this parser is written in Ruby and has it’s own DSL, which is documented at the link that I provided earlier.
+
+Make your parser look like this:
+
+
+```ruby
+class OtagoHocken < SupplejackCommon::Oai::Base
```
Comment (motizuki) [inline _posts/2017-12-04-example-project.md]: nice example 💃

File: _posts/2017-12-04-example-project.md [hunk 2]
```diff
@@ -0,0 +1,469 @@
...
+
+### Step Seven - Indexing your Records
+
+Once the harvest has finished you are now ready to index your records into Solr. On our DNZ servers we have a background job that handles this, but since we are running locally we can just do it in the Rails console of the API.
+
+Cd into your api repository, enter the rails console and run the following.
+
+```Ruby
+Sunspot.session = Sunspot::Rails.build_session
+SupplejackApi::Record.all.map(&:index)
```
Comment (motizuki) [inline _posts/2017-12-04-example-project.md]: You can run `Sunspot.index(Record.all)` which will only perform 1 call to Solr.

**Review by motizuki — APPROVED** [review]
🥇

File: _posts/2017-12-04-example-project.md (hunk 1 above)
Comment (richardmatthewsdev) [reply _posts/2017-12-04-example-project.md]: Yeah it's a good one from Dan :)

File: _posts/2017-12-04-example-project.md (hunk 2 above)
Comment (richardmatthewsdev) [reply _posts/2017-12-04-example-project.md]: Cool I will update!

**Pull request merged**